projects/igniteui-angular/src/lib/core/styles/components/grid/_grid-theme.scss

```diff
@@ -699,7 +699,7 @@
     }
 
     %grid-row--mrl {
-        border-bottom: none !important;
+        border-bottom: 1px solid transparent;
```

Comment:
This doesn't work because of the selector specificity, there is a double border between each row.

Try this
```
%grid-row--mrl {
  &%grid-row {
      border-bottom-color: transparent;
  }
}
```

Reply:
You will also need 
```
%igx-grid__tr--edit {
 &%grid-row {
     border-bottom: 1px solid --var($theme, 'edit-mode-color');
 }
}
```